authors/apps/articles/models.py

```diff
@@ -71,10 +71,25 @@ class Comment(models.Model):
     def json(self):
 
         return {
```

Comment:
I find this block of code not being used in any way, I believe using a renderer would be more efficient.

Reply:
Thanks for spotting it. I have removed the block of code and made the necessary updates to the code.